### PR TITLE
fixes to SEMS conversion for order of either-neither and measure titles

### PR DESCRIPTION
**Pull request by benadida**
ensure either-neither contests appear in the right order, and fix title of measures to have a colon instead of a newline.

### CODE REVIEW

File: converter/SEMSinput.py [hunk 1]
```diff
@@ -131,10 +133,17 @@ def process_row(row):
...
 
-        values = ["?"] * len(table_def["fields"])
-        sql = "insert into %s values (%s)" % (table_def['name'], ",".join(values))
+        fields = table_def["fields"]
+        value_placeholders = ["?"] * len(fields)
 
-        c.execute(sql, row[1:])
+        sql = "insert into %s values (%s)" % (table_def['name'], ",".join(value_placeholders))
+
+        values = row[1:]
```
Comment (eventualbuddha) [inline converter/SEMSinput.py:141]:
```suggestion
        id, *values = row
```

**Comment by benadida** [comment]
> The placeholder stuff feels a little like a band-aid, but okay for now. It may warrant a refactor in the future.

agreed 100%. We need `array.splice`!

File: converter/SEMSinput.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ def process_row(row):
 
         sql = "insert into %s values (%s)" % (table_def['name'], ",".join(value_placeholders))
 
-        values = row[1:]
+        id, *values = row
```
Comment (eventualbuddha) [inline converter/SEMSinput.py:141]: You need to actually use `id` in place of `row[0]` too. Sorry, I wasn't explicit about that.

Comment (benadida) [reply converter/SEMSinput.py:141]: I think it works either way, but you're right that this will be cleaner, so will do that now.

Comment (eventualbuddha) [reply converter/SEMSinput.py:141]: It does work either way, but I wrote the initial comment to improve the clarity of the parts of `row`.

Comment (benadida) [reply converter/SEMSinput.py:141]: yep, totally agreed :)

**Pull request merged**